pynumdiff/tests/test_utils.py

```diff
@@ -73,7 +73,7 @@ def identity(x, dt): return x, 0 # should come back the same
     x = np.arange(100)
     kernel = utility.gaussian_kernel(9)
 
-    x_hat, dxdt_hat = utility.slide_function(identity, x, 0.1, kernel, step_size=2)
+    x_hat, dxdt_hat = utility.slide_function(identity, x, 0.1, kernel, stride=2)
```

Comment:
I decided to rename this parameter. I like `stride` better, because `dt` is a step size. Less ambiguous.